src/VPolytope.jl

```diff
@@ -322,27 +322,29 @@ Compute the convex hull of the set union of two polytopes in V-representation.
 
 - `P1`         -- polytope
 - `P2`         -- another polytope
-- `backend`    -- (optional, default: `default_polyhedra_backend(P1, N)`) the polyhedral
-                  computations backend, see [Polyhedra's documentation](https://juliapolyhedra.github.io/Polyhedra.jl/latest/installation.html#Getting-Libraries-1)
-                  for further information
+- `backend`    -- (optional, default: `nothing`) the polyhedral
+                  computations backend
 
 ### Output
 
 The `VPolytope` obtained by the concrete convex hull of `P1` and `P2`.
 
 ### Notes
 
+This function takes the union of the vertices of each polytope and then relies
+on a concrete convex hull algorithm. For low dimensions, a specialied implementation
+for polygons is used. For higher dimensions, `convex_hull` relies on the polyhedral
+computations backend specified in the `backend` keyword argument, which defaults
+to `default_polyhedra_backend(P1, N)`.
```

Comment:
You just changed it to `nothing`. Maybe just remove the comment about the default.